src/test/java/io/lettuce/core/cluster/pubsub/RedisClusterPubSubConnectionIntegrationTests.java

```diff
@@ -100,33 +96,25 @@ void testRegularClientPubSubChannels() {
         assertThat(channelsOnOtherNode).isEmpty();
     }
 
-    @Test
-    void testRegularClientPubSubShardChannels() {
```

Comment:
lets have this test not removed since its the only test that aims `pubsubShardChannels` command. you can uncomment ssubscribe commands as todo lines suggested, ssubscribe implementation is done.

Reply:
Sure, because both tests used the PUBSUB SHARDCHANNELS command I assumed its a duplication. But it makes sense to have one test only for this scenario. Added back.